ArjunaJTS/standalone/pom.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <!--
     JBoss, Home of Professional Open Source
-    Copyright 2014, Red Hat, Inc. and/or its affiliates, and individual
```

Comment:
I think this line should be left with the year the file was created and not changed to `2020`. At least it usual way how we do this.

Reply:
+1